pkg/utils/syscalls/syscalls-helper.go

```diff
@@ -0,0 +1,33 @@
+package syscalls
+
+// Copyright 2023 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+func GetSyscallNumberByName(name string) (int, bool) {
+	var number int
+	var ok bool
+
+	number, ok = syscallsNameToNumber[name]
+
+	return number, ok
```

Comment:
nit: (same in GetSyscallNameByNumber)

```suggestion
	return syscallsNameToNumber[name]
```

Reply:
This doesn't work for some reason, it unpacks only the value but not the "ok" var unless you do it as I do it.

Reply:
You're right. 

Perhaps 

```	
	number, ok := syscallsNameToNumber[name]
	return number, ok
``` 

is a bit more concise?